runtime/doc/options.txt

```diff
@@ -6922,7 +6922,9 @@ A jump table for the options with a short description can be found at |Q_op|.
 'writedelay' 'wd'	number	(default 0)
 			global
 	The number of milliseconds to wait for each character sent to the
-	screen.  When non-zero, characters are sent to the terminal one by
-	one.  For debugging purposes.
+	screen.  When positive, characters are sent to the terminal one by
+	one.  When negative, all redrawn characters cause a delay, even if the
+	character already was displayed by the terminal.  For debugging
```

Comment:
should "terminal" be replaced with "UI" in this help?

Reply:
:+1: